packages/fx-core/src/common/samples.ts

```diff
@@ -41,7 +41,7 @@ class SampleProvider {
             url:
               (sample as any).relativePath && (sample as any).url
                 ? (sample as any).url
-                : sampleConfigV3.baseUrl,
+                : `${sampleConfigV3.baseUrl}/${sample.id}`,
```

Comment:
```suggestion
                : `${sampleConfigV3.baseUrl}${sample.id}`,
```
The baseUrl already has a '/' at the end.

Reply:
Nice catch! I wasn't aware of this since opening is successful with additional '/'.